public/app/features/dashboard/components/PanelEditor/OptionsPaneItemDescriptor.tsx

```diff
@@ -28,7 +29,7 @@ export class OptionsPaneItemDescriptor {
   constructor(public props: OptionsPaneItemProps) {}
 
   getLabel(searchQuery?: string): ReactNode {
-    const { title, description, overrides } = this.props;
+    const { title, description, overrides, addon } = this.props;
```

Comment:
I would like to avoid this. in the end, the `render` function allows to render anything

Reply:
Did this approach grow on you @ivanortegaalba? 😬